package.json

```diff
@@ -73,6 +73,11 @@
   },
   "resolutions": {
     "@testing-library/dom": "^9.0.0",
-    "@types/react": "18.2.74"
+    "@types/react": "18.2.74",
+    "@storybook/blocks@^7.6.17": "patch:@storybook/blocks@npm%3A7.6.17#./.yarn/patches/@storybook-blocks-npm-7.6.17-481e2fff02.patch",
+    "@storybook/blocks@7.6.17": "patch:@storybook/blocks@npm%3A7.6.17#./.yarn/patches/@storybook-blocks-npm-7.6.17-481e2fff02.patch"
```

Comment:
```suggestion
    "@storybook/blocks@^7.6.17": "patch:@storybook/blocks@npm%3A7.6.17#./.yarn/patches/@storybook-blocks-npm-7.6.17-481e2fff02.patch",
```

?

Reply:
yarn did this automatically, so i'd actually quite like to leave it like this _just_ incase.